acceptance/bundle/variables/double_underscore/output.txt

```diff
@@ -0,0 +1,13 @@
+
+>>> $CLI bundle validate
+Error: incorrect variable name: [${var.double__underscore}]
```

Comment:
Let's add a reason why the variable name is malformed. Can be hard to notice otherwise.